librz/core/tui/panels.c

```diff
@@ -1953,6 +1954,14 @@ static bool __handle_mouse_on_X(RzCore *core, int x, int y) {
 }
 
 static bool __handle_mouse_on_panel(RzCore *core, RzPanel *panel, int x, int y, int *key) {
+	// mouse click contains down and up
+	// thus use on_mouse_down to avoid repeated process
+	static bool on_mouse_down = false;
```

Comment:
We advise against using static variables and global state:
- https://github.com/rizinorg/rizin/issues/1874
- https://github.com/rizinorg/rizin/issues/276

Reply:
Is there a way to distinguish between the up and down mouse events?